src/extension/gdal/src/main/java/org/geoserver/gdal/GDALStatus.java

```diff
@@ -68,9 +68,13 @@ public boolean isEnabled() {
         return Optional.ofNullable("");
     }
 
-    public String getGDALWrapperJarVersion() {
+    public Optional<String> getGDALWrapperJarVersion() {
         if (isAvailable()) {
-            return GeoTools.getVersion(gdal.class).toString();
+            Version v = GeoTools.getVersion(gdal.class);
+            if (v == null) {
+                return Optional.empty();
```

Comment:
@DBlasby - The jar is just called "gdal.jar". I'm fairly sure the NPE was coming from the toString() call.